tests/network-tests/src/scenarios/content-directory.ts

```diff
@@ -1,15 +1,23 @@
 import createAndUpdateChannel from '../flows/clis/createAndUpdateChannel'
 import commentsAndReactions from '../flows/content/commentsAndReactions'
+import videoCategories from '../flows/content/videoCategories'
 import curatorModerationActions from '../flows/content/curatorModerationActions'
+import activeVideoCounters from '../flows/content/activeVideoCounters'
 import nftAuctionAndOffers from '../flows/content/nftAuctionAndOffers'
 import leadOpening from '../flows/working-groups/leadOpening'
 import { scenario } from '../Scenario'
 
 // eslint-disable-next-line @typescript-eslint/no-floating-promises
 scenario('Content directory', async ({ job }) => {
   const leadSetupJob = job('Set content working group leads', leadOpening(true, ['contentWorkingGroup']))
+
+  // following jobs must be run sequentially due to some QN queries that could interfere
   const channelJob = job('Create and Update Channel with assets', createAndUpdateChannel).requires(leadSetupJob)
-  job('nft auction and offers', nftAuctionAndOffers).requires(channelJob)
-  job('curator moderation actions', curatorModerationActions).requires(channelJob)
-  job('video comments and reactions', commentsAndReactions).after(channelJob)
+  const videoCategoriesJob = job('video categories', videoCategories).after(channelJob)
+  const videoCountersJob = job('check active video counters', activeVideoCounters).requires(videoCategoriesJob)
+  const nftAuctionAndOffersJob = job('nft auction and offers', nftAuctionAndOffers).after(videoCountersJob)
+  const curatorModerationActionsJob = job('curator moderation actions', curatorModerationActions).after(
+    nftAuctionAndOffersJob
+  )
+  job('video comments and reactions', commentsAndReactions).after(curatorModerationActionsJob)
```

Comment:
`curatorModerationActionsJob` does not really need `nftAuctionAndOffersJob` to be completed first. So they can be run in parallel. I think the current approach unnecessarily increases the running time of tests
```suggestion
  job('nft auction and offers', nftAuctionAndOffers).after(videoCategoriesJob)
  job('curator moderation actions', curatorModerationActions).after(videoCategoriesJob)
  job('video comments and reactions', commentsAndReactions).after(videoCategoriesJob)
```